<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Research | Firas Khatib</title>

    <meta name="viewport" content="width=1000, initial-scale=1.0, maximum-scale=1.0">

    <!-- Loading Bootstrap -->
    <link href="bootstrap/css/bootstrap.css" rel="stylesheet">

    <!-- Loading Flat UI -->
    <link href="css/flat-ui.css" rel="stylesheet">
    <link href="css/fkhatib.css" rel="stylesheet">
    

    <link rel="shortcut icon" href="images/Retina-Ready.ico">

    <!-- HTML5 shim, for IE6-8 support of HTML5 elements. All other JS at the end of file. -->
    <!--[if lt IE 9]>
      <script src="js/html5shiv.js"></script>
      <script src="js/respond.min.js"></script>
    <![endif]-->
	
  </head>
  <body>
		<header>
			<nav id="navbar" class="navbar navbar-default navbar-fixed-top" role="navigation">
			  <div class="container-fluid">
				<!-- Brand and toggle get grouped for better mobile display -->
				<div class="navbar-header">
				  <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
					<span class="sr-only">Toggle navigation</span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				  </button>
				  <a class="navbar-brand" href="index.html"><p class="nav-name">Khatib's Lab</p></a>
				</div>

				<!-- Collect the nav links, forms, and other content for toggling -->
				<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
				  
				  <ul class="nav navbar-nav navbar-right">
					<li><a href="index.html" data-loc="research">Research</a></li>
					<li><a href="publications.html" data-loc="publications">Publications</a></li>
					<li><a href="media.html" data-loc="media">Media</a></li>
					<li><a href="contact.html" data-loc="contact">Contact Information</a></li>
					<li><a href="cv.html" data-loc="cv">CV</a></li>
					
				  </ul>
				</div><!-- /.navbar-collapse -->
			  </div><!-- /.container-fluid -->
			</nav>
		</header>
		
	<div class="container">
		<section class="scroll-panel">
		<div class="jumbotron jumbotron-midnight">
			<h5 class="">Prediction and Design of Protein Structures and Protein-protein Interactions</h5>
			<p><span style=""><strong><span style="font-size: medium; font-family: helvetica;">From prediction of structure to design of function</span></strong></span></p>
			<p><span style="font-size: small; font-family: helvetica;">&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; The primary goals of the research in the Baker group over the past several years have been to predict the structures of naturally occurring biomolecules and interactions and to design new molecules with new and useful functions. These prediction and design challenges have direct relevance for biomedicine and provide stringent and objective tests of our understanding of the fundamental underpinnings of molecular biology.</span><br /><a class="research_links" href="prediction of structure to design of function.html">Read More</a></p>
			<h6><span style="font-size: medium; "><strong><span style="font-family: helvetica;">Design of New Protein Functions </span></strong></span></h6>
			<p><span style="font-size: small; font-family: helvetica;">&nbsp;&nbsp; &nbsp; &nbsp; &nbsp; Several years ago, we developed a general computational strategy for designing new protein structures that incorporates full backbone flexibility into rotamer-based sequence optimization. This was accomplished by integrating ab initio protein structure prediction, atomic-level energy refinement, and sequence design in Rosetta. The procedure was used to design Top7, a 93-residue protein with a novel sequence and topology. Top7 was found to be folded and highly stable, and the x-ray crystal structure of Top7 is virtually identical &nbsp;to the design model (Kuhlman et al, 2003). </span><br /><a class="research_links" href="design of new protein functions.html">Read More</a><span style="font-size: small; font-family: helvetica;">&nbsp;&nbsp; &nbsp; &nbsp; &nbsp; </span></p>
			<h6><span style="font-size: medium; "><strong><span style="font-family: helvetica;">Design of Endonucleases with New DNA-Cleavage Specificites </span></strong></span></h6>
			<p><span style="font-size: small; font-family: helvetica;">&nbsp;&nbsp; &nbsp; &nbsp; &nbsp; In the past several years, we have extended the Rosetta protein design methodology to protein-RNA and protein-DNA interfaces and shown that new, highly specific endonucleases can be created by redesign of the extended DNA-binding interface in homing endonucleases (Havranek et al, 2004; Ashworth et al, 2006). We are continuing to improve this methodology and designing new endonucleases that cleave within therapeutically important sites. For gene therapy applications, for example, we are designing endonucleases that cleave near the sites of mutations that cause disease; our collaborators will then experiment with correcting mutations in these genes through homologous recombination by introducing the designed endonuclease and a wild-type copy of the gene into mutant cells.&nbsp; Thus far, we have successfully designed endonucleases with a range of cleavage specificities, and are getting close to the goal of producing enzymes which specifically cleave physiological target sites to go into gene therapy trials (Ashworth et al, 2006; Thyme et al, 2009).</span><br /><span style="font-size: small; font-family: helvetica;">Tradeoff between experiment and computation</span><br /><a class="research_links" href="design of endonucleases with new DNA-cleavage specificites.html">Read More</a><span style="font-size: small; font-family: helvetica;"></span></p>
			<h6><span style="font-size: medium; "><strong><span style="font-family: helvetica;">Plans for the Future </span></strong></span></h6>
			<p><span style="font-size: small; font-family: helvetica;">&nbsp;&nbsp; &nbsp; &nbsp; &nbsp; We will continue to work to improve the physical model and the sampling methodology underlying the prediction and design calculations in Rosetta. On the structure calculation side, we will strive for consistent near-atomic resolution ab initio structure prediction for small proteins, and work towards atomic level structure determination for proteins greater than 200 amino acids using limited experimental data such as backbone only NMR data and 5-7Å electron density data.&nbsp;&nbsp; We will focus in particular on membrane proteins and other systems for which obtaining high resolution experimental data is difficult—this is where our approach are likely to contribute the most.&nbsp; We will also extend data guided structure determination to biological assemblies where SAXS, crosslinking and other types of data often can be collected.&nbsp; On the design side, we will extend our methodology to non natural amino acids and cofactors to try to leapfrog over the limitations nature has faced with the limited set of twenty amino acids.&nbsp; We are aiming to design a complete pathway for fuel production from CO2 using solar generated reducing equivalents.&nbsp; We will also develop and test methods for designing high affinity binders/inhibitors for any specified surface patch on a protein of known structure.&nbsp; More generally, &nbsp;we hope to develop new biomolecules with new functions—inhibitors, enzymes, endonucleases, and vaccines—that can have a positive impact on the world.</span><br /><span style="font-size: small; font-family: helvetica;">&nbsp;<img src="images/figure1.jpg" /></span></p>
			<p><span style="font-size: small; font-family: helvetica;"><span style="">Figure 1</span>. Examples of design models for experimentally validated de novo designed enzymes.&nbsp; The chemical reactions catalyzed by the designed enzymes are indicated below the structure schematics in the black panels.</span></p>
		</div>
		</section>
	</div>	
    
	
	<footer id="footer" class="footer-height footer-color">
      <div class="container footer-row">
		<div class="footer-left">
		<br/>For more information:<br/>

		
		<a href="http://www.umassd.edu/engineering/cis/people/facultyandstaff/firaskhatib/" class="linkcolor info">UMass Dartmouth</a>
		<a href="http://scholar.google.com/citations?user=1WSaBBsAAAAJ" class="linkcolor info"></br/>Google Scholar Page</a><br/>
		</div>
        <div class="row footer-right">
			<div class="col-xs-7">
				<a target="_blank" href="http://fold.it/portal/info/about" class="footer-brand">
				  <img class="ftr-img" src="images/fold-it.png" />
				</a>
			</div> 
        </div>
      </div>
    </footer>
    
	<!-- Load JS here for greater good =============================-->
    <script src="js/jquery-1.8.3.min.js"></script>
    <script src="js/jquery.scrollTo.js"></script>	
    <script src="js/jquery-ui-1.10.3.custom.min.js"></script>
    <script src="js/jquery.ui.touch-punch.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="js/bootstrap-select.js"></script>
    <script src="js/bootstrap-switch.js"></script>
    <script src="js/flatui-checkbox.js"></script>
    <script src="js/flatui-radio.js"></script>
    <script src="js/jquery.tagsinput.js"></script>
    <script src="js/jquery.placeholder.js"></script>
    <script src="js/jquery.stacktable.js"></script>
    <script src="http://vjs.zencdn.net/4.3/video.js"></script>
    <script src="js/application.js"></script>
	
	<script>
		(function ($) {
			$('#navbar, #bottom-nav').on('click', 'li a', function(){
				var $clickedAnchor = $(this);
					$panelToScrollTo = $('#' + $clickedAnchor.data('loc'));
				//console.log($panelToScrollTo[0]);
			});
			
		}(jQuery));
	</script>
	
  </body>
</html>
